files/docker/hooks/build

```diff
@@ -1,3 +1,7 @@
 #!/bin/bash
 
+# required because this script is executed in Dockerfile location instead of provided
+# build context location, which is root for $DOCKERFILE_PATH
```

Comment:
I've read it a couple of times and I'm still struggling with understanding the context.
Wouldn't it help if we moved this script in the "Dockerfile location", i.e. to `files/docker/`?